tico/experimental/quantization/ptq/utils/reduce_utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+
+def reduce_except(x: torch.Tensor, dim_to_keep: int):
```

Comment:
```suggestion
def reduce_minmax_except(x: torch.Tensor, dim_to_keep: int):
```

Reply:
Nice catch. I'll rename it to `channelwise_minmax` for more readability.